modules/ldsg.js

```diff
@@ -77,33 +87,47 @@ async function processCardAction(int) {
       suggestionIcarus: u.sf.channels.bottestingForum
     }[int.customId];
     if (channel) {
-      embed.addFields({ name: `Sent to <#${channel}>`, value: `by ${int.user}` });
+      embed.addFields({ name: `Status`, value: `Sent to <#${channel}> by ${int.user}` });
       int.client.getForumChannel(channel)?.threads.create({ name: `Suggestion from ${suggestion.embeds[0].author?.name}`, message: { embeds: [suggestion.embeds[0]], components: replyOption } });
       return int.update({ embeds: [embed], components: [] });
     } else if (int.customId == "suggestionReply") {
       await int.showModal(modal);
       submitted = await int.awaitModalSubmit({ time: 5 * 60 * 1000, dispose: true }).catch(() => {
         return null;
       });
-      if (!submitted) {
-        return int.channel?.send("I fell asleep waiting for your input...");
-      }
+      if (!submitted) return int.channel?.send("I fell asleep waiting for your input...");
       await submitted.deferUpdate();
       reply = submitted.fields.getTextInputValue("text");
       em = u.embed({ author: int.user })
           .setTitle("Suggestion feedback")
           .addFields({ name: "Reply:", value: reply });
-      member = int.guild.members.cache.get(suggestion.embeds[0].footer?.text.slice(9) ?? "");
-      if (!member) {
-        return int.channel?.send("I could not find that member");
-      }
+      member = int.guild.members.cache.get(suggestion.embeds[0].footer?.text ?? "");
+      if (!member) return int.channel?.send("I could not find that member");
       try {
         member.send({ embeds: [em] });
         return int.channel?.send({ content: "Replied to user:", embeds: [em] });
       } catch (e) {
         u.errorHandler(e, int);
         return int.channel?.send("Failed to message member, they may have me blocked. You will need to reach out to them on your own this time!");
       }
+    } else if (int.customId == "suggestionRename") {
+      if(!int.channel) return int.reply("Channel error");
+      const post = int.guild.channels.cache.get(int.channel.id);
```

Comment:
this seems a bit redundant. you should probably do a isThread check on the channel though.